src/main/resources/servers/verily-autopush.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "verily-autopush",
-  "clientCredentialsFile": "verily_secret.json",
+  "clientCredentialsFile": "verily_auth0_dev_secret.json",
```

Comment:
clarifying:
- dev secret for - dev, autopush and staging
- prod secret for - prod and preprod